src/test/CurrencyTest.sol

```diff
@@ -17,7 +17,7 @@ contract CurrencyTest {
     }
 
     function isNative(Currency currency) external pure returns (bool) {
-        return currency.isNative();
+        return currency.isAddressZero();
     }
```

Comment:
this test file and its unit tests are the only places where `isNative()` is used in any capacity, so we should remove it entirely

Reply:
`isNative` is used in the pool manager?

Reply:
its not, pulled it locally and searched for it